docs/reference/licensing/update-license.asciidoc

```diff
@@ -79,16 +79,16 @@ POST /_license
 NOTE: These values are invalid; you must substitute the appropriate content
 from your license file.
 
-You can alternatively use a `curl` command, for example:
+You can also install your license file using a `curl` command. Be sure to add
+`@` before the license path.
```

Comment:
Just a suggestion. Not sure if this is _too_ much information for this, but personally knowing why I need to add switch to a command, helps me remember it

```suggestion
`@` before the license file path to instruct curl to treat it as an input file.
```